tests/tools/MatchBin.cc

```diff
@@ -795,6 +795,41 @@ TEST_CASE("Test MatchBin", "[tools]")
   REQUIRE(bitBin64.GetTags(bitBin64.Match(bs9, 5)) == (emp::vector<emp::BitSet<64>> {bs9, bs1, bs7}));
 
   }
+  // test ImprintRegulators
+  {
+    emp::Random rand(1);
+    emp::MatchBin<
+      std::string,
+      emp::AbsDiffMetric,
+      emp::RouletteSelector<>,
+      emp::LegacyRegulator
+    > bin1(rand);
+
+    const size_t hi1 = bin1.Put("hi", 1);
+    const size_t bye1 = bin1.Put("bye", 2);
+
+    bin1.SetRegulator(hi1, 0.1);
+    bin1.SetRegulator(bye1, 0.2);
+
+    REQUIRE(bin1.GetRegulator(hi1).state == 0.1);
+    REQUIRE(bin1.GetRegulator(bye1).state == 0.2);
+
+    emp::Random rand2(1);

```

Comment:
comment: set up matchbin to imprint on template

Reply:
added on 798bdf6